### DISCUSSION TITLE
Releases or at least builds?

### DISCUSSION THREAD

**Discussion by eobet** [miroslavpejic85/p2p#23] [General]
This looks very interesting, but are there any plans to provide an executable?

I'm guessing none is provided to avoid it spreading too far in this early stage, though...

I think I managed to build it using Visual Studio (I'm not a developer but I have it installed), however when I try to run the executable, I get this:

![image](https://user-images.githubusercontent.com/3254196/162951843-d0c2659a-573f-4c48-b653-c220b0b3e4ac.png)

**Comment by miroslavpejic85** [comment]
Hello @eobet, 
While not a developer and managed to compile it, you were great. 
It's a `false positive detection`  (app not known by av). You can found some reply about in the closed issues. I don't release the executable because I want people to trust the app and compile the code for them such as it's written and transparent to all in the repo.
Thanks for the understanding.
Cheers.

**Reply by aquaphx** [reply]
I can understand the reason but the problem is I'm really noob, I don't even know how to begin =(

**Comment by miroslavpejic85** [comment]
Hello @aquaphx,
1. Download the Project from the git
2. Download  the IDE: https://visualstudio.microsoft.com/vs/ (Community 2022) 
3. Open it (p2pconn.sln) & Run - Compile it with a click.

**Comment by aquaphx** [comment]
thank you, I'll try it out 😊

On Wed, Jul 20, 2022 at 8:18 PM Miroslav Pejic ***@***.***>
wrote:

> Hello @aquaphx <https://github.com/aquaphx>,
>
>    1. Download the Project from the git
>    2. Download the IDE: https://visualstudio.microsoft.com/vs/ (Community
>    2022)
>    3. Open it (p2pconn.sln) & Run - Compile it with a click.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/miroslavpejic85/p2p/discussions/23#discussioncomment-3189064>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AGUKLE6ZHOKK7BV55TMV3JTVU74CBANCNFSM5TG7NUKA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
*Best Regards And Thank You.*

Eko Juwana
08991634476 (Whatsapp)
https://lazyelf.shop
FB & IG :
fb.me/lazyelfshop
instagram.com/lazyelfshop

**Reply by aquaphx** [reply]
@miroslavpejic85 wow I got headaches to be honest lol.. the download is too big.. and also there's some thing that I must download again..
I don't find the "run" I think it's the "Play / Start" ? 
but anyway I press "Build" and found 2 folder inside the "bin" which is "Debug" and "Release", sorry to trouble you again, which one should I use ? 
and if i want to share it to the computer I want to remote desktop with, do I have to include the (dll, xml, config and pdb) ?

**Comment by miroslavpejic85** [comment]
You need to sue Microsoft for a IDE installation that caused your headache, It's not a p2p related issue :)

![p2p-remote-desktop-portable](https://user-images.githubusercontent.com/70838434/180436010-58889112-75c5-4cd7-8a21-0316d9a75ba6.png)

After the Build, in the bin/Release It is enough only `Newtonsoft.Json.dll` that has been recently added to be able to write to the [StunServers.json](https://github.com/miroslavpejic85/p2p/issues/21) file (list of Stun Servers editable from the app) that will be created at the first start in the same dir + `p2p.exe`.

Being portable, put it in a USB stick and run it from wherever you want as needed.
Regards.

**Reply by aquaphx** [reply]
haha yeah I think I should sue, It gave me so many options and a lot of downloads lol (ehem.. just like windows 10 👿)
Thank you for your patient, I'm zipping it and will trying it out.
But most of all, thank you for this excellent app. 🙏😊

**Reply by miroslavpejic85** [reply]
Well, don't worry, when I can gladly help. Have a nice weekend.

**Comment by aquaphx** [comment]
Thank you, have a great weekend as well 😊
